polyfill/lib/calendar.mjs

```diff
@@ -1688,6 +1695,7 @@ const helperJapanese = ObjectAssign(
     // The last 3 Japanese eras confusingly return only one character in the
     // default "short" era, so need to use the long format.
     eraLength: 'long',
+    erasBeginMidYear: true,
```

Comment:
I know we found the problem in Japanese, but does this only affect Japanese?  Or do other calendars need this behavior too? Would it be harmful to make all calendars behave the same?

Reply:
I believe the only calendar affected is Japanese, but this is part of the calendar-dependent part of the spec anyway, so it wouldn't be a normative change if we wanted to tweak it in our reference code (unless it was covered later in the era and month codes proposal).